webapp/navigation.py

```diff
@@ -26,9 +26,17 @@ def get_current_page_bubble(path):
 
     page_bubbles = copy.deepcopy(secondary_navigation_data)
 
+    parent = None
     for page_bubble_name, page_bubble in page_bubbles.items():
         if path.startswith(page_bubble["path"]):
             current_page_bubble = page_bubble
+            if parent is None:
+                parent = page_bubble.get("parent", None)
```

Comment:
```suggestion
    for page_bubble_name, page_bubble in page_bubbles.items():
        if path.startswith(page_bubble["path"]):
            current_page_bubble = page_bubble
            parent = page_bubble.get("parent", None)
```
I think you can get rid of these two lines (I haven't tested this)